### PR TITLE
x11-terms/terminology: bump to 1.2.0

### PR DESCRIPTION
**Pull request by juippis**
Simple bump and switch from enlightenment.eclass to meson.eclass. Seems to build and run fine...

Also, add myself as proxy-maintainer for this package.
Also, updated HOMEPAGE.

Closes: https://bugs.gentoo.org/631580
Package-Manager: Portage[mgorny]-2.3.33.1

### CODE REVIEW

**Comment by mgorny** [comment]
Looks clean and simple. Thank you! I'm just going to test and merge.

**Comment by mgorny** [comment]
```
ERR<15694>:edje lib/edje/edje_calc.c:706 _edje_real_part_image_set() Error type: EVAS_LOAD_ERROR_UNKNOWN_FORMAT
## Copy & Paste the below (until EOF) into a terminal, then hit Enter

eina_btlog << EOF
/usr/lib64/libeina.so.1	 0x7f25d6896f19 0x7f25d686f000
/usr/lib64/libeina.so.1	 0x7f25d6897ee3 0x7f25d686f000
/usr/lib64/libeina.so.1	 0x7f25d6899329 0x7f25d686f000
/usr/lib64/libedje.so.1	 0x7f25d70a6cfe 0x7f25d7079000
/usr/lib64/libedje.so.1	 0x7f25d70ad46c 0x7f25d7079000
/usr/lib64/libedje.so.1	 0x7f25d70b0bd4 0x7f25d7079000
/usr/lib64/libevas.so.1	 0x7f25d6061ba5 0x7f25d5fce000
/usr/lib64/libevas.so.1	 0x7f25d6062aec 0x7f25d5fce000
/usr/lib64/libevas.so.1	 0x7f25d60b5bf3 0x7f25d5fce000
/usr/lib64/libevas.so.1	 0x7f25d60b927b 0x7f25d5fce000
/usr/lib64/libevas.so.1	 0x7f25d6032ece 0x7f25d5fce000
/usr/lib64/ecore_evas/engines/x/v-1.20/module.so	 0x7f25c6ddf7bf 0x7f25c6dd6000
/usr/lib64/libecore_evas.so.1	 0x7f25d5b5d5c1 0x7f25d5b4a000
/usr/lib64/libecore.so.1	 0x7f25d5d8c5ee 0x7f25d5d73000
/usr/lib64/libeo.so.1	 0x7f25d3d21393 0x7f25d3d0e000
/usr/lib64/libeo.so.1	 0x7f25d3d1dc8c 0x7f25d3d0e000
/usr/lib64/libecore.so.1	 0x7f25d5d8fba6 0x7f25d5d73000
/usr/lib64/libecore.so.1	 0x7f25d5d901ef 0x7f25d5d73000
/usr/bin/terminology	 0x559f26dc4cff 0x559f26da7000
/usr/bin/terminology	 0x559f26dbbf4c 0x559f26da7000
/lib64/libc.so.6	 0x7f25d41590b1 0x7f25d4138000
/usr/bin/terminology	 0x559f26dbbf8a 0x559f26da7000
EOF
```

I suppose you're missing some USE-dependencies on `dev-libs/efl`.

**Comment by mgorny** [comment]
(yes, I've built it with `USE="-* X"` ;-))

**Comment by juippis** [comment]
Welp. I could reproduce that problem.
I dug up terminology's [meson.build](https://github.com/billiob/terminology/blob/master/meson.build#L46) and found that 'eet' USE flag is indeed required. However, it didnt work with `"-* eet X"` so after intense session of trial-and-error type of testing I came up with this line: 
`USE="-* eet fontconfig opengl X" emerge -av1 efl` - can you test it before I update this PR? 

I also spotted [option for nls](https://github.com/billiob/terminology/blob/master/meson_options.txt#L8) and will add it when updating PR.

**Comment by mgorny** [comment]
Yeah, that seems to work. Also:

```
CRI<18341>:eina_log ../terminology-1.2.0/src/bin/win.c:1756 win_new() unknown log domain -1, original message level was: WRN
ERR<18341>:termpty ../terminology-1.2.0/src/bin/termpty.c:274 _fd_read_do() error while reading from tty slave fd: Input/output error
## Copy & Paste the below (until EOF) into a terminal, then hit Enter

eina_btlog << EOF
/usr/lib64/libeina.so.1	 0x7f23543e4f19 0x7f23543bd000
/usr/lib64/libeina.so.1	 0x7f23543e5ee3 0x7f23543bd000
/usr/lib64/libeina.so.1	 0x7f23543e7329 0x7f23543bd000
/usr/bin/terminology	 0x55eed478ba86 0x55eed474e000
/usr/lib64/libecore.so.1	 0x7f23538dccf9 0x7f23538c0000
/usr/lib64/libecore.so.1	 0x7f23538dd1ef 0x7f23538c0000
/usr/bin/terminology	 0x55eed476bcff 0x55eed474e000
/usr/bin/terminology	 0x55eed4762f4c 0x55eed474e000
/lib64/libc.so.6	 0x7f2351ca60b1 0x7f2351c85000
/usr/bin/terminology	 0x55eed4762f8a 0x55eed474e000
EOF

ERR<18341>:ecore_con lib/ecore_con/ecore_con.c:144 ecore_con_shutdown() Ecore_Con Shutdown called without calling Ecore_Con Init.

## Copy & Paste the below (until EOF) into a terminal, then hit Enter

eina_btlog << EOF
/usr/lib64/libeina.so.1	 0x7f23543e4f19 0x7f23543bd000
/usr/lib64/libeina.so.1	 0x7f23543e5ee3 0x7f23543bd000
/usr/lib64/libeina.so.1	 0x7f23543e7329 0x7f23543bd000
/usr/lib64/libecore_con.so.1	 0x7f23529e625d 0x7f23529cb000
/usr/lib64/libecore_file.so.1	 0x7f2353491a21 0x7f235348e000
/usr/lib64/libefreet_mime.so.1	 0x7f23509740a2 0x7f2350971000
/usr/lib64/libeio.so.1	 0x7f2350b86854 0x7f2350b77000
/usr/lib64/libelementary.so.1	 0x7f2354828394 0x7f235463c000
/usr/lib64/libelementary.so.1	 0x7f235482850a 0x7f235463c000
/usr/bin/terminology	 0x55eed476b51b 0x55eed474e000
/usr/bin/terminology	 0x55eed4762f4c 0x55eed474e000
/lib64/libc.so.6	 0x7f2351ca60b1 0x7f2351c85000
/usr/bin/terminology	 0x55eed4762f8a 0x55eed474e000
EOF

mgorny@pomiot ~/git/gentoo $ ERR<18403>:ecore_con lib/ecore_con/ecore_con.c:144 ecore_con_shutdown() Ecore_Con Shutdown called without calling Ecore_Con Init.

## Copy & Paste the below (until EOF) into a terminal, then hit Enter

eina_btlog << EOF
/usr/lib64/libeina.so.1	 0x7fedbc250f19 0x7fedbc229000
/usr/lib64/libeina.so.1	 0x7fedbc251ee3 0x7fedbc229000
/usr/lib64/libeina.so.1	 0x7fedbc253329 0x7fedbc229000
/usr/lib64/libecore_con.so.1	 0x7fedba9cd25d 0x7fedba9b2000
/usr/lib64/libecore_file.so.1	 0x7fedbc023a21 0x7fedbc020000
/usr/lib64/efreet/v-1.20/efreet_mime_cache_create	 0x5604a2a9ecb5 0x5604a2a9d000
/lib64/libc.so.6	 0x7fedbb3ae0b1 0x7fedbb38d000
/usr/lib64/efreet/v-1.20/efreet_mime_cache_create	 0x5604a2a9f7fa 0x5604a2a9d000
EOF


mgorny@pomiot ~/git/gentoo $ ERR<18618>:ecore_con lib/ecore_con/ecore_con.c:144 ecore_con_shutdown() Ecore_Con Shutdown called without calling Ecore_Con Init.

## Copy & Paste the below (until EOF) into a terminal, then hit Enter

eina_btlog << EOF
/usr/lib64/libeina.so.1	 0x7f8f29b77f19 0x7f8f29b50000
/usr/lib64/libeina.so.1	 0x7f8f29b78ee3 0x7f8f29b50000
/usr/lib64/libeina.so.1	 0x7f8f29b7a329 0x7f8f29b50000
/usr/lib64/libecore_con.so.1	 0x7f8f282f425d 0x7f8f282d9000
/usr/lib64/libecore_file.so.1	 0x7f8f2994aa21 0x7f8f29947000
/usr/lib64/libefreet.so.1	 0x7f8f292a0fa9 0x7f8f2929a000
/usr/lib64/efreet/v-1.20/efreet_desktop_cache_create	 0x5605d7c6768a 0x5605d7c64000
/lib64/libc.so.6	 0x7f8f28cd50b1 0x7f8f28cb4000
/usr/lib64/efreet/v-1.20/efreet_desktop_cache_create	 0x5605d7c6774a 0x5605d7c64000
EOF
```

But I suppose it's normal ;-P.

File: x11-terms/terminology/terminology-1.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="nls"
+
+DEPEND="
+	>=dev-libs/efl-1.20.0[eet,fontconfig,opengl,X]
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )
+"
```
Comment (mgorny) [inline x11-terms/terminology/terminology-1.2.0.ebuild]: You're missing RDEPEND now.

**Comment by mgorny** [comment]
Please open a rekeywording request for ppc/ppc64.

**Pull request closed without merging**